boards/qn9080dk/Kconfig

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2020 iosabi
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+config BOARD
+    default "qn9080dk" if BOARD_QN9080DK
+
+config BOARD_QN9080DK
+    bool
+    default y
+    select BOARD_COMMON_QN908X
+    select CPU_MODEL_QN9080XHN
+
+    # Put defined MCU peripherals here (in alphabetical order)
+    select BOARD_HAS_XTAL
+    #select BOARD_XTAL
```

Comment:
yeah, I actually wanted BOARD_XTAL_32M here, but it wasn't possible because it was a choice. I made those now a regular config so you can set either one from the board.